br-ext/package/optee_examples/optee_examples.mk

```diff
@@ -14,7 +14,8 @@ define OPTEE_EXAMPLES_BUILD_TAS
 			$(MAKE) CROSS_COMPILE="$(shell echo $(BR2_PACKAGE_OPTEE_EXAMPLES_CROSS_COMPILE))" \
 			O=out TA_DEV_KIT_DIR=$(OPTEE_EXAMPLES_SDK) \
 			$(TARGET_CONFIGURE_OPTS) -C $${f%/*} all; \
-	done
+			if [ $$? != 0 ]; then _failed=1; break; fi; \
```

Comment:
I'm wondering if `@(set -e; for ... )` couldn't be used instead. The `()` is to have a sub-shell so the `set -e` doesn't spread, this may not be needed depending on how `OPTEE_EXAMPLES_BUILD_TAS` is evaluated.

Reply:
Good suggestion, works fine. I think the parentheses are needed, based on [the way Buildroot uses this fragment](https://github.com/buildroot/buildroot/blob/95942f5fcd/package/pkg-generic.mk#L242).